tests/unit/models/transactions/test_account_delete.py

```diff
@@ -0,0 +1,46 @@
+from unittest import TestCase
+
+from xrpl.models.exceptions import XRPLModelException
+from xrpl.models.transactions import AccountDelete
+
+_ACCOUNT = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_DESTINATION = "rf7HPydP4ihkFkSRHWFq34b4SXRc7GvPCR"
+
+
+class TestAccountDelete(TestCase):
+    MAX_CREDENTIALS_ERROR = (
```

Comment:
Why is this a separate class variable when it's only being used once?

Reply:
This was a suggestion by coderabbit AI to improve the readability. I think it does help with the readability.

Reply:
I would have it be a helper function in the test where it's used, rather than in the class, then.

Reply:
Do you mean to say that I need to restrict the scope of the error message? Declare it within the method?

Reply:
2d86ea29